src/hue_flask.py

```diff
@@ -45,9 +48,9 @@ def set_color():
 
     if color_name == "random":
         is_random = True
-        color_sum = int(database.get('color_sum').decode('utf-8'), base=10)
+        color_sum = 160 #int(database.get('color_sum').decode('utf-8'), base=10)
```

Comment:
I had color_sum in the redis database because it could change in the future if a color is added.  It doesn't work at the current moment so we need to fix it.